assets/base.css

```diff
@@ -2227,22 +2231,39 @@ product-info .loading-overlay:not(.hidden) ~ *,
   line-height: calc(1 + 0.1 / var(--font-body-scale));
 }
 
-/* section-announcement-bar */
-.announcement-bar {
+/* utility-bar */
+.utility-bar {
+  height: 100%;
+}
+
+.utility-bar--bottom-border {
   border-bottom: 0.1rem solid rgba(var(--color-foreground), 0.08);
+}
+
+.announcement-bar,
+.announcement-bar-slider__announcement {
   color: rgb(var(--color-foreground));
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  align-content: center;
+}
+
+.announcement-bar-slider,
+.announcement-bar-slider .slider {
+  width: 100%;
 }
 
 .announcement-bar__link {
```

Comment:
I'm aligned with the decision to remove the background hover effect, but now not sure if the arrow animation alone is enough feedback to visually indicate the hover state. I wonder if it might be worth reducing the initial opacity of the text and changing to 100% on hover, like many other links do ([example](https://screenshot.click/29-57-q7tlj-x8jos.mp4))

Not sure if this was considered or not, but wanted to run it by @YoannJailin just in case.

Reply:
Hey! Agree an additional effect could help. Did some testing, the only matter i see with opacity is that some announcements may be clickable, some others not. Might look strange to switch between annoucements more or less transparent depending of they are clickable or not. 

Is it worth considering a line on hover when it's clickable? 
<img width="1219" alt="Capture d’écran, le 2023-03-30 à 14 22 58" src="https://user-images.githubusercontent.com/95925370/228834550-f8d37428-d4a3-45fe-8c70-1bd15ba8ca91.png">

Reply:
@YoannJailin done ✅